public/js/views/del-pay-method.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component, PropTypes } from 'react';
+
+import Modal from 'components/modal';
+import CardChoice from 'components/card-choice';
+
+import { default as tracking } from 'tracking';
+import { gettext } from 'utils';
+
+
+export default class DelPayMethod extends Component {
+
+  static propTypes = {
+    closeModal: PropTypes.func.isRequired,
+    delCreditCard: PropTypes.func.isRequired,
+    payMethods: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.number,
+        resource_uri: PropTypes.string,
+        truncated_id: PropTypes.string,
+        type_name: PropTypes.string,
+      })
```

Comment:
Yeah, I was thinking that too - it gets passed down through several components. Seems a lot of repetition for nothing.